src/pages/ShoppingCart.page.js

```diff
@@ -24,4 +39,75 @@ export class ShoppingCartPage extends BaseSwagLabPage {
     async removeCartItemById(id) {
         await this.cartItems.nth(id).locator(this.removeItemSelector).click();
     }
+
+    //verify cart page
+    async verifyCartPage(){
+        await this.page.waitForURL('**/cart.html')
+        await expect(this.page.getByText('Your Cart')).toBeVisible();
+    }
+
+    //check selected item in cart
+    async checkSelectedItem(itemArray){
+        await expect(this.page.locator('.cart_item').first().locator('.inventory_item_name')).toHaveText(itemArray[0].name);
+        await expect(this.page.locator('.cart_item').nth(1).locator('.inventory_item_name')).toHaveText(itemArray[1].name);
```

Comment:
Its weird to expect .first() and .nth(1) to equal itemArray[0] and itemArray[1]. Lets move this logic from function to test itself aaaand show the logic, why do we pick first and nth(1).